helper/helper.go

```diff
@@ -136,25 +136,34 @@ func (h *Helper) Get(serverURL string) (string, string, error) { // nolint: gocy
 			h.logger.Info("no cached token(s) were read. Re-authenticating.")
 		}
 
+		var validTokens []string
```

Comment:
Would you mind changing this line to

```go
validTokens := make([]string, 0, len(cachedTokens))
```
We try to preallocate our slices. Otherwise looks great.